routes/index.routes.js

```diff
@@ -2,6 +2,29 @@ const express = require('express');
 const router = express.Router();
 
 /* GET home page */
+
 router.get('/', (req, res, next) => res.render('index'));
 
+router.get('/main', (req, res) => {
```

Comment:
Once you're use the block helpers in the views to not show contect if you're not logged it, you don't need this code here. Nevertheless, the "private-pages" routes should be, in my point of view, inside the auth.routes.js.